iosApp/iosApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,62 +6,62 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/google/abseil-cpp-binary.git",
       "state" : {
-        "revision" : "748c7837511d0e6a507737353af268484e1745e2",
-        "version" : "1.2024011601.1"
+        "revision" : "194a6706acbd25e4ef639bcaddea16e8758a3e27",
+        "version" : "1.2024011602.0"
       }
     },
     {
       "identity" : "app-check",
```

Comment:
question (non-blocking): Can this dependency be removed now, or does it get pulled in by default with firebase?

Reply:
I did remove AppCheck as a dependency from the app so it seems like this is just getting pulled in by default now